packages/fxa-settings/src/lib/hooks.tsx

```diff
@@ -55,3 +59,21 @@ export function useChangeFocusEffect() {
 
   return elToFocus;
 }
+
+// Creates a relier state based on URL query parameters and state of local/session stage.
+const urlContext = new UrlContext(window);
```

Comment:
We could also maybe just do this setup as part of the AppContext... I didn't look to closely at since I know we were talking about using hooks.

Reply:
@LZoog I think this is pretty clean now. The relier factory is added to the app context, and then there is a small hook that makes it easy to get the current relier from that factory via `useRelier`.